File: Data Structures & Algorithms/duplicate-integer/submission-0.java
class Solution {
    public boolean hasDuplicate(int[] nums) {
        for(int i=0;i<nums.length;i++)
        {
            int ii=nums[i];
            for(int j=i+1;j<nums.length;j++){
                if(ii==nums[j])
                {
                    return true;
 
                }
             
            }
        }
        return false;
    }
}

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.cpp
class Solution {
public:
    int longestConsecutive(vector<int>& nums) {
        unordered_set<int> s;

        for(int num : nums)
        {
            s.insert(num);
        }

        int longest = 0;

        for(int num : s)
        {
            if(s.count(num - 1) == 0)
            {
                int current = num;
                int length = 1;

                while(s.count(current + 1))
                {
                    current++;
                    length++;
                }

                longest = max(longest, length);
            }
        }

        return longest;
    }
};